glide.yaml

```diff
@@ -43,6 +43,10 @@ import:
     version: v1.2.2
     repo:    https://github.com/aws/aws-sdk-go
 
+### Rackspace
+  - package: github.com/rackspace/gophercloud
+    ref:     42196eaf5b93739d335921404bb7c5f2205fceb3
```

Comment:
Hey @akutz for your last comment about the glide.yaml file, I found the particular commit in question:
https://github.com/rackspace/gophercloud/commit/42196eaf5b93739d335921404bb7c5f2205fceb3
but I have no idea how to render the information you asked for- when I ported it, I simply reused the old RR 3.3 glide as a template